mars-cli/mars_lib/models/repository_response.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import List, Optional
+from pydantic import BaseModel, field_validator, Field, ConfigDict
+import pydantic
+import pydantic.alias_generators
+from mars_lib.target_repo import TargetRepository
+
+
+class Filter(BaseModel):
+    key: str
+    value: str
+
+
+class Path(BaseModel):
+    key: str
+    where: Optional[Filter] = None
+
+
+class Accession(BaseModel):
+    path: List[Path] = []
+    value: str
+
+    @field_validator("path")
+    def validate_path(cls, path: List[Path]) -> List[Path]:
+        keys = [p.key for p in path]
+        if len(keys) != len(set(keys)):
+            raise ValueError("Duplicate keys found in path list")
+        return path
+
+
+class Error(BaseModel):
+    type: str
+    message: str
+    path: List[Path] = []
+
+
+class Info(BaseModel):
+    name: str
```

Comment:
As we discussed, the `name` field will probably be optional unless we can think of a good use-case for it.

Reply:
Understood! I adjusted it to an optional field in the model :-)
Thanks for reviewing!